### ISSUE TITLE
[FEATURE] Use humantime instead of milliseconds for time values

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/wasmCloud#3257]
### Affected project(s)

- [ ] documentation
- [ ] examples
- [X] wasmCloud host
- [X] wasmCloud CLI (wash)
- [ ] wasmCloud dashboard UI (washboard)
- [ ] capability providers
- [ ] provider bindgen
- [ ] control interface client
- [ ] other / not sure

### Is your feature request related to a problem? Please describe.

For `wash` and `wasmcloud`, we have many timeout/delay/execution limit time values that are stated in milliseconds. This is fine for smaller values on the order of thousands, but when you start to think about max execution time in minutes this gets unwieldy (e.g. MAX_EXECUTION_TIME_MS=600000)

### Describe the solution you'd like

I'd like to deprecate the flags that take timeout values in wasmCloud and wash and replace them with [humantime](https://crates.io/crates/humantime) parseable values. So instead of supplying "600000" milliseconds for 10 minutes (which you could still do with 600000ms) you could simply supply "10m"

For wasmCloud it's very important this is done in a backwards compatible way, and we retain the older flags (and hide them) until we release v2.0.0. For wash flags can be deprecated in one minor release and removed in the next.

**Comment by omarjatoi** [comment]
👋 just indicating my intent to work on this issue. I started collecting all the places where the `wash` cli is currently using a non-humantime value, and will do the same for `wasmcloud` host. Should have a PR up in a few days!

**Comment by vados-cosmonic** [comment]
Hey @omarjatoi how do you feel about adding the new options that use humantime *alongside* the existing `-ms` extensions? The non-`-ms` flags can reasonably expect humantime (and fail otherwise)

**Comment by vados-cosmonic** [comment]
Yeah that first approach sounds reasonable -- if we can re-use the function that does parsing for both that would be great.

Some tests would go a long way in making sure we get the exact functionality you're thinking about as well.

**Comment by omarjatoi** [comment]
Sounds good. Thanks @vados-cosmonic!

**Comment by vados-cosmonic** [comment]
No worries! Thanks for checking in -- feel free to let us know if you have any questions or run into anything else !

**Comment by brooksmtownsend** [comment]
Just chiming in to say what you're planning and working on @omarjatoi looks perfect 😄 Thank you so much for the thoughtful approach to this issue!

**Comment by omarjatoi** [comment]
I just opened https://github.com/wasmCloud/wasmCloud/pull/3440 as a WIP PR. It currently only addresses wasmCloud host, but this is generally what I'll be doing for `wash` as well.

**Comment by omarjatoi** [comment]
Not stale, work in #3440 just somewhat delayed (busier than anticipated past several weeks)

**Comment by omarjatoi** [comment]
Hi @jtakakura, please go ahead! Sorry about dropping the ball on this!

**Comment by vados-cosmonic** [comment]
The project has changed so drastically that this idea is probably not reasonable to pursue in this form -- closing this as we're focusing effort now on wasmCloud v2.

If you think this definitely still applies towards v2, please feel free to open a new issue!

(I expect the PR related to this will be closed soon if it hasn't been already!)
